clusters/app.ci/supplemental-ci-images/snyk-cli/snyk-cli.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: snyk-cli
+  namespace: ci
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  namespace: ci
+  name: snyk-cli
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
+      name: snyk-cli:latest
+      namespace: ci
+  source:
+    dockerfile: |
+      FROM origin-release:golang-1.18
```

Comment:
Why do we need golang? snyk requires it?

Reply:
Correct. `snyk-deps` will invoke `go list` internally to list dependencies

Reply:
Can we use this image (to keep the building env consistent)?

https://github.com/openshift/ci-tools/blob/master/.ci-operator.yaml

Reply:
I'm actually not sure about that. All we need is an image with `golang 1.18`. But since `snyk-cli` will be used outside of `ci-tools`, is it appropriate to use `ci-tools-build-root`...?

Reply:
IMHO, if the tool to check things need go, then it should be the same go env as we build our binaries.
I am even thinking of adding `snyk` into the build root image.
https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/ci-tools-build-root.yaml

Reply:
I see. I will switch to `ci-tools-build-root`